wandb/sdk/artifacts/artifact.py

```diff
@@ -61,6 +61,7 @@
 from wandb.sdk.internal.thread_local_settings import _thread_local_api_settings
 from wandb.sdk.lib import filesystem, retry, runid, telemetry
 from wandb.sdk.lib.hashutil import B64MD5, b64_to_hex_id, md5_file_b64
+from wandb.sdk.lib.mailbox import MailboxHandle
```

Comment:
Nit: this is only used for type checking, move it to line 74, and add quotation marks on line 150.

Reply:
fixed